### ISSUE TITLE
Incorporate avt e2e tests in our PR checks

### ISSUE THREAD

**Issue by matthewgallo** [carbon-design-system/ibm-products#4425]
## What will this achieve?

This will add an additional layer of a11y testing for the components in our library. We should be able to follow a similar approach to Carbon in their `ci.yml`, see [here](https://github.com/carbon-design-system/carbon/blob/main/.github/workflows/ci.yml#L183-L241).

## How will success be measured?

When an avt PR check is included that runs the e2e accessibility tests.

## Additional information

- Designs
- Existing code
- etc
